podman/Containerfile

```diff
@@ -22,7 +24,9 @@ ADD /base_images/* "$AI_PATH/base_images/"
 ADD /cache_images/* "$AI_PATH/cache_images/"
 ADD /systemd_banish.sh "$AI_PATH/"
 WORKDIR "$AI_PATH"
-RUN bash ./podman/setup.sh
+RUN cd $AI_PATH && \
+    chmod +x ./podman/setup.sh && \
+    $AI_PATH/podman/setup.sh
```

Comment:
Is the mix-and-match of absolute and relative paths necessary? `s/$AI_PATH/./` in line 29 might make it more readable.

Reply:
Oh woops, I intended to fix that.  Thanks for pointing it out.